script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,z) coordinates in a CSV-format list
```

Comment:
We can certainly make it optional. For querying, the example you point assumes we have a raster of elevation that we're querying.

However, we may be in luck that the USGS National Elevation Dataset has an API:
http://ned.usgs.gov/epqs/

Here is an example query that returns elevation for XY (-76.116081,42.794448):
http://ned.usgs.gov/epqs/pqs.php?x=-76.116081&y=42.794448&units=Meters&output=xml
Elevation can be feet or meters.